python/lsst/ts/aos/utils/m1m3/M1M3LUT.py

```diff
@@ -104,39 +112,50 @@ async def update_lut_force_balance(
         start_time,
         end_time,
     )
+    print("Elevations\n", elevations["actualPosition"].describe())
+    expected_count = interval_ms / 70.0
```

Comment:
should this 70 be a parameter? Maybe add a comment specifying where this 70 comes from? As in "70ms is the time interval at which the elevation is samples"